pims/norpix_reader.py

```diff
@@ -50,7 +62,8 @@ class NorpixSeq(FramesSequence):
     This is the native format of StreamPix software, owned by NorPix Inc.
     The format is described in the StreamPix documentation.
 
-    Currently supports uncompressed 8-bit monochrome files only.
+    Currently supports uncompressed files only, either
+    uint8/16/32 monochrome or as_raw color or monochrome.
```

Comment:
The "uint" reference could be confusing for casual users, since StreamPix does not use C or numpy names for datatypes. How about "8, 16, or 32 bits per pixel"?

Also, the purpose of `as_raw` might not be obvious from this docstring — that it is essentially a workaround that allows the user to do her own processing of the frame data. It would be good to elaborate on that a bit, to avoid confusion over whether color images are "officially" supported.